databases/gcp-mysql-ssl/README.md

```diff
@@ -0,0 +1,25 @@
+# GCP MySQL with SSL
+
+Google Cloud MySQL with SSL
+
+Prisma docs: https://www.prisma.io/docs/concepts/database-connectors/mysql#configuring-an-ssl-connection
+GCP docs: https://cloud.google.com/sql/docs/mysql/configure-ssl-instance
+
+## How to run this locally
+
+### Environment variables
+
+The environment variable `GCP_MYSQL_SSL_DB_URL` should point to a GCP MySQL database. It should follow the following pattern:
+
+`mysql://<user>:<password>@<host>:<port>/<database>?sslmode=require&sslcert=../server-ca.pem&sslidentity=../client-identity.p12&sslpassword=<client-identity-password>&sslaccept=accept_invalid_certs`
+
+The 3 mentioned files `server-ca.pem`, `client-cert.pem`, `client-key.pem` are to be downloaded from GCP and put into the root directory next to this `README.md` file. (In the tests of this project, these files are created on demand from environment variables that contain their content)
```

Comment:
This is still not telling the full story. Downloading the files and putting them there, will miss the execution of the necessary command, which comes from the documentation.

A suggestion again:

```suggestion
The environment variable `GCP_MYSQL_SSL_DB_URL` should point to a GCP MySQL database. It should follow the following pattern to include the SSL configuration:

`mysql://<user>:<password>@<host>:<port>/<database>?sslmode=require&sslcert=../server-ca.pem&sslidentity=../client-identity.p12&sslpassword=<client-identity-password>&sslaccept=accept_invalid_certs`

The referenced files `server-ca.pem` and `client-identity.p12` are created from 3 files (`server-ca.pem`, `client-cert.pem`, `client-key.pem`) that are downloaded from GCP and put into the root directory next to this `README.md` file. (In the tests of this project, these files are created on demand from environment variables that contain their content, then the additional file is created via the `run.sh` script.)
```

Reply:
Not anymore, since I changed the script at the bottom of the readme to call `sh run.sh`, it calls the required command.

Reply:
Replicated the same in Pg test